### DISCUSSION TITLE
which directory our own Chatflow strored

### DISCUSSION THREAD

**Discussion by shrutichauhan12** [FlowiseAI/Flowise#515] [Q&A]
Hi,
As we know, our flowise setup is stored under npm folder. but I didn't get the folder where our own created chatflow stored.
can someone help me with this.

**Comment by HenryHengZJ** [comment]
its stored in your home directory. For example: `C:\Users\Henry\.flowise`

Here's the [code](https://github.com/FlowiseAI/Flowise/blob/main/packages/server/src/DataSource.ts#L12)

You can also change where its being stored by specifying the env variable `DATABASE_PATH=your-path`

**Comment by chungyau97** [comment]
Linux: `/home/your_name/.flowise`
